examples/DayPickerRangeControllerWrapper.jsx

```diff
@@ -122,7 +124,17 @@ class DayPickerRangeControllerWrapper extends React.Component {
   }
 
   onDatesChange({ startDate, endDate }) {
-    this.setState({ startDate, endDate });
+    const { daysViolatingMinNightsCanBeClicked, minimumNights } = this.props;
+    let doesNotMeetMinNights = false;
+    if (daysViolatingMinNightsCanBeClicked && startDate && endDate) {
+      const dayDiff = endDate.diff(startDate.clone().startOf('day').hour(12), 'days');
+      doesNotMeetMinNights = dayDiff < minimumNights && dayDiff >= 0;
+    }
+    if (doesNotMeetMinNights) alert('That day does not meet the minimum nights requirement');
```

Comment:
this needs to be removed before merging

Reply:
@ljharb just curious why this needs to be removed?

Reply:
oh oops, i didn't realize this was an example; so that makes it not blocking, but `alert` blocks the main JS thread so it shouldn't ever be used in any scenario, including examples.

Reply:
@ljharb updated to show the message in the calendar info instead